CHANGES.rst

```diff
@@ -4,12 +4,23 @@ Change Log
 Unreleased
 ----------
 
+**Added**
+
+* :class:`.Reddit` keyword argument ``refresh_token_manager``.
+
 **Deprecated**
 
-* :meth:`.me` will no longer return ``None`` when called in :attr:`.read_only` mode 
+* The configuration setting ``refresh_token`` is deprecated and its use will result in a
+  :py:class:`DeprecationWarning`. This deprecation applies in all ways of setting
+  configuration values, i.e., via ``praw.ini``, as a keyword argument when initializing
+  an instance of :class:`.Reddit`, and via the ``PRAW_REFRESH_TOKEN`` environment
+  variable. To be prepared for PRAW 8, use the new :class:`.Reddit` keyword argument
+  ``refresh_token_manager``. See "Working with Refresh Tokens" in PRAW's documentation
```

Comment:
Should be able to link to the section in the docs.

Reply:
Yes, absolutely. I haven't cleaned up all these references yet.